packages/libs/react-components/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, SystemIcons } from './../../';
+import { CardColor } from './styles';
+import { Card, ICardProps } from '.';
+import { CardBody, CardColors, CardFooter, CardHeading } from './';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    color: CardColor;
+    expand: boolean;
+  } & ICardProps
+> = {
+  title: 'Layout/Card',
+  argTypes: {
+    color: {
+      options: Object.values(CardColors),
```

Comment:
I would expect this to get the options from the variant object like the other stories do rather than a CardColors type. Is there a reason why you decided to do it this way?